mutiny-wasm/src/lib.rs

```diff
@@ -1705,6 +1705,14 @@ impl MutinyWallet {
         let npub = parse_npub_or_nip05(&npub).await?;
         Ok(npub.to_bech32().expect("bech32"))
     }
+
+    /// If the invoice is from a node that gives hodl invoices
+    #[wasm_bindgen]
+    pub async fn is_potential_hodl_invoice(invoice: String) -> Result<bool, MutinyJsError> {
```

Comment:
Why is this needed? Don't we already warn users when paying zeus that it is a hodl invoice? Something is already doing that logic somewhere based on the list of pubkeys.

Reply:
In the front end we only warn for lightning address by checking for @zuespay.com